cmake/Platform/Common/Install_common.cmake

```diff
@@ -91,6 +91,10 @@ function(ly_setup_target OUTPUT_CONFIGURED_TARGET ALIAS_TARGET_NAME absolute_tar
         cmake_path(RELATIVE_PATH target_library_output_directory BASE_DIRECTORY ${CMAKE_LIBRARY_OUTPUT_DIRECTORY} OUTPUT_VARIABLE target_library_output_subdirectory)
     endif()
 
+    string(APPEND archive_output_directory "/${PAL_PLATFORM_NAME}/$<CONFIG>/${BUILD_PERMUTATION}")
+    string(APPEND library_output_directory "/${PAL_PLATFORM_NAME}/$<CONFIG>/${BUILD_PERMUTATION}")
+    string(APPEND runtime_output_directory "/${PAL_PLATFORM_NAME}/$<CONFIG>/${BUILD_PERMUTATION}")
```

Comment:
We should use [cmake_path(APPEND)](https://cmake.org/cmake/help/v3.21/command/cmake_path.html?highlight=cmake_path#append) here